README.md

```diff
@@ -82,6 +82,7 @@ The following devices are supported by this version of liquidctl.  See each guid
 | AIO liquid cooler | [NZXT Kraken Z63, Z73](docs/kraken-x3-z3-guide.md) | USB & USB HID | <sup>_E_</sup> |
 | DDR4 DRAM | [Corsair Vengeance RGB](docs/ddr4-guide.md) | SMBus | <sup>_EUXN_</sup> |
 | DDR4 DRAM | [DIMMs with a standard temperature sensor](docs/ddr4-guide.md) | SMBus | <sup>_EUXN_</sup> |
+| Fan/LED controller | [Corsair Commander Pro](docs/corsair-commander.md) | <sup>_EN_</sup> |
```

Comment:
Missing value for column "bus" ("USB  HID").

Reply:
ah, I had thought that was some sort of replacement column for `<sup>_EN_</sup>` and was confused by it. But I just added that in.